example/cumulus-tf/discover_and_queue_pdrs_with_execution_name_prefix_workflow.tf

```diff
@@ -2,7 +2,7 @@ module "discover_and_queue_pdrs_with_execution_name_prefix_workflow" {
   source = "../../tf-modules/workflow"
 
   prefix          = var.prefix
-  name            = "DiscoverAndQueuePdrsWithExecutionNamePrefix"
+  name            = "DiscoverAndQueuePdrsExecutionPrefix"
```

Comment:
I explicitly left the more descriptive name for the TF object and the workflow definition, as I think it provides context.